packages/account/src/Components/poi/status/unsupported/unsupported.jsx

```diff
@@ -19,12 +20,18 @@ const checkNimcStep = documents => {
 
 const Unsupported = ({ country_code, ...props }) => {
     const [detail, setDetail] = React.useState(null);
-    const toggleDetail = index => setDetail(index);
+    const toggleDetail = index => {
```

Comment:
@likhith-deriv do we need curly braces here? we can leave it as one-liner as before, right? 🙂

Reply:
Will remove them